app/assets/stylesheets/image_x.scss

```diff
@@ -2,3 +2,67 @@
 @import 'modules/media';
 @import 'vendor/tooltip';
 
+.#{$namespace}-image-x {
+  height: 400px;
+  position: relative;
+}
+
+.#{$namespace}-image-x-thumb-slider-container {
+  background: rgba(0, 0, 0, 0.5);
+  border: 1px solid #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.